main.css

```diff
@@ -0,0 +1,3 @@
+.my-header {
+  color: red;
+}
```

Comment:
- [ ] Nice job adding styles using CSS, however, if you take a look at your project you can see that the text "Hello Microverse" doesn't appear red :red_circle: , this can be because you aren't linking your .css file to your .html file. 

![image](https://user-images.githubusercontent.com/28109626/147309925-d7e9def8-b5b8-41b3-87c7-a599343c8a8a.png)

Please let's fix that